tee-worker/litentry/core/credentials/src/templates/a4.json

```diff
@@ -0,0 +1,55 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/ed25519-2020/v1"
+    ],
+    "id": "",
+    "type": [
+        "VerifiableCredential"
+    ],
+    "issuer": {
+        "id": "",
+        "shard": "",
+        "name": ""
+    },
+    "subject": "",
+    "issuanceBlockNumber": 0,
+    "expirationBlockNumber": 0,
+    "credentialSubject": {
+        "id": "",
+        "description": "Identity Linked And Verified",
+        "type": "IdentityLinkedVerified",
```

Comment:
please change the above `description` and `type`, e.g, to `Check whether any of the linked accounts hold a minimum amount of LIT NOW` and `LIT Holder` or something else.

Reply:
fixed